research-software-directory/RSD-as-a-service
# 236 & 237 project count and find ror id

# Project count on organisation overview and ror id find in organisation settings

Closes #236 
Closes #237 

Changes proposed in this pull request:
* Organisation overview page shows organisations ordered by score (SUM(software,projects,teams))
* Organisation card:
   * Project count is added to organisation card
   * Verified "badge" (icon) is added for tenant organisations (is_tenant=true)
   * Placement of organisation logo is improved
* Find ROR id button is added to organisation settings, next to ror id input. If single organisation is matched by name ror_id is loaded to input. If no exact match on organisation name is possible, the modal with returned options is shown for user to select organisation.

How to test:
* `docker-compose down --volumes`: to remove old
* `docker-compose build` to build
* `docker-compose up`: to start
* perform data migration, to be able to see organisation cards
* make yourself maintainer of all/some organisation (otherwise you won't be able to test #237
* navigate to organisation overview, the cards should have software and project counts
* select organisation you made yourself a maintainer
* navigate to settings and use update button next tot ror id input:
  * when organisation can be matched exactly by name the ror_is will be added to input form
  * when organisation cannot be matched exactly by name, but api returns options, the modal will organisations will be shown to user to select organisation as ror_id "donor".

## Example organisation overview
![image](https://user-images.githubusercontent.com/9204081/174452889-d8fc4f3d-e9d9-4b8f-bc0b-3a788169e475.png)

## Example find ror id in organisation settings, and icon buttons for logo change
![image](https://user-images.githubusercontent.com/9204081/174452954-0a826bac-4598-4482-992b-4af0961b3880.png)

PR Checklist:
